requirements-doc.txt

```diff
@@ -0,0 +1,6 @@
+sphinx
+ipython
+matplotlib
+sphinx_rtd_theme
+doctr
```

Comment:
One small note here...is it necessary to add this to requirements-doc? Contribs don't need to install to work on documentation. It is only needed by Travis. Perhaps we could remove from here and install explicitly in CI

Reply:
I specifically separated to allow "text" handling of documentation build dependencies.

Reply:
This file should go into docs/requirements for RTD compat.